.travis.yml

```diff
@@ -10,20 +19,17 @@ before_install:
     - export PATH=~/miniconda3/bin/:$PATH
     - conda update --yes conda
 install:
-    - conda config --add channels bioconda
-    - conda create -n testenv --yes python=$TRAVIS_PYTHON_VERSION pip
-    - source activate testenv
-    - conda install --yes --file conda_requirements.txt
-    - pip install -r requirements.txt
-    - pip install .
+    - pip3 install -r requirements.txt 
+    - pip3 install pytest flake8 coverage
+    - pip3 install .
 script:
     - make coverage
     - make lint
```

Comment:
```
script:
    - make coverage
    - make lint
```

Both of these can be accomplished with pytest plugins (e.g. pytest-cov and pytest-flake8). This might make your testing and CI process more straightforward.